openpype/plugins/publish/extract_burnin.py

```diff
@@ -676,6 +698,9 @@ def prepare_repre_data(self, instance, repre, burnin_data, temp_data):
         # Add representation name to burnin data
         burnin_data["representation"] = repre["name"]
 
+        burnin_frame_range = temp_data["frame_range"]
+        keep_frame_index = instance.data.get("keepFrameIndex", burnin_frame_range)
```

Comment:
Why the fallback value is not a bool but a value, seems like a bug

Reply:
If no custom_frames is given, temp_data["frame_range"] = []
And if the instance doesn't have a "keepFrameIndex" info (which is a bool), it will retrieve the burnin_frame_range

So keep_frame_index can either be a bool, or an empty list or a list with element.

This way, any "if keep_frame_index" would work with a list which may be empty or not or a bool.

It's a security to ensure that the frame Id display in the burIn would correspond to the same frame index in tvpp in case custom_frame is given but keep_frame_index is off, if keep_frame_index is off in publish, the data "keepFrameIndex" won't exists in the instance data dict

Reply:
Seems a bit hacky that a value named like a bool can also store a list, not a fan of that

Reply:
Yes me too
I'm on it !

Reply:
Ok, I've discovered that all those part is useless, because the burnin_frame_start and burnin_frame_end are correctly set previously throught extract sequence
So we don't have to check for keep_index or custom_range ^^'